benchmark-analyze-results.py

```
@@ -1,36 +1,48 @@
 
-    old_file = 'benchmark-results/old-results.json'
-    new_file = 'benchmark-results/new-results.json'
 
-    print("🔍 ЗАГРУЗКА РЕЗУЛЬТАТОВ С ПРАВИЛЬНОЙ ФИЛЬТРАЦИЕЙ")
-    print("=" * 60)
+def parse_args():
+    parser = argparse.ArgumentParser(description='Анализ результатов JMH-бенчмарков')
+    parser.add_argument('--old', default='benchmark-results/old-results.json',
+                        help='Путь к JSON с результатами старой версии')
+    parser.add_argument('--new', default='benchmark-results/new-results.json',
+                        help='Путь к JSON с результатами новой версии')
```

Ruff (0.15.18)

[warning] String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)